### PR TITLE
clang-format code formatting

### PR DESCRIPTION
**Pull request by puszupek**

### CODE REVIEW

File: src/api/session/ui/Imp-base.cpp [hunk 1]
```diff
@@ -351,18 +351,18 @@ auto UI::Imp::blockchain_selection(
...
-                        opentxs::factory::BlockchainSelectionModel(
-                            api_, chains, cb)))
-                .first;
+                  .emplace(
+                      std::piecewise_construct,
+                      std::forward_as_tuple(type),
+                      std::forward_as_tuple(
+                          opentxs::factory::BlockchainSelectionModel(
+                              api_, chains, cb)))
+                  .first;
```
Comment (bartalke) [inline src/api/session/ui/Imp-base.cpp]: Instead of that you can try to use try_emplace, it should be more readable then :P

**Review by bartalke — APPROVED** [review]

File: src/api/session/ui/Imp-base.cpp (hunk 1 above)
Comment (puszupek) [reply src/api/session/ui/Imp-base.cpp]: I can but it will be not only clang-format anymore.

Comment (puszupek) [reply src/api/session/ui/Imp-base.cpp]:
```
    auto it =
        blockchain_selection_
            .try_emplace(
                type,
                opentxs::factory::BlockchainSelectionModel(api_, chains, cb))
            .first;
```

Comment (puszupek) [reply src/api/session/ui/Imp-base.cpp]: Is it the equivalent?

**Review by TomaszAugustyn — APPROVED** [review]

File: src/api/session/ui/Imp-base.cpp (hunk 1 above)
Comment (bartalke) [reply src/api/session/ui/Imp-base.cpp]: clang-format cannot complain if you want to use specific function :P 
No, I would say:
auto it = blockchain_selection_.try_emplace(type, api_, chains, cb).first;

Comment (puszupek) [reply src/api/session/ui/Imp-base.cpp]: It doesn't work because BlockchainSelectionModel returns unique_ptr. My proposition was compiled.

**Comment by puszupek** [comment]
Final merge was made in a mirror PR:
https://github.com/matterfi/opentxs/pull/161

**Pull request closed without merging**